### PR TITLE
Added resize API for the terminal renderer

### PR DESCRIPTION
**Pull request by javierdlg**
Added resizing API for the terminal renderer

### CODE REVIEW

File: src/cascadia/PublicTerminalCore/HwndTerminal.cpp [hunk 1]
```diff
@@ -384,3 +384,18 @@ void _stdcall SetTheme(void* terminal, TerminalTheme theme, LPCWSTR fontFamily,
...
+HRESULT _stdcall Resize(void* terminal, unsigned int rows, unsigned int columns)
+{
+    auto publicTerminal = reinterpret_cast<HwndTerminal*>(terminal);
+    COORD newSize = { 0, 0 };
+
+    HRESULT result = publicTerminal->_renderEngine->GetFontSize(&newSize);
+    newSize.Y *= (short) rows;
+    newSize.X *= (short) columns;
+
+    result = publicTerminal->_terminal->UserResize(newSize);
```
Comment (ZoeyR) [inline src/cascadia/PublicTerminalCore/HwndTerminal.cpp]: I thought `UserResize` took a COORD describing the row and column height, but here it looks like you're passing in pixel heights.

File: src/cascadia/PublicTerminalCore/HwndTerminal.hpp [hunk 2]
```diff
@@ -19,7 +19,8 @@ extern "C"
 	__declspec(dllexport) HRESULT _stdcall CreateTerminal(HWND parentHwnd, _Out_ void** hwnd, _Out_ void** terminal);
 	__declspec(dllexport) void _stdcall SendTerminalOutput(void* terminal, LPCWSTR data);
 	__declspec(dllexport) void _stdcall RegisterScrollCallback(void* terminal, void __stdcall callback(int, int, int));
-	__declspec(dllexport) HRESULT _stdcall TriggerResize(void* terminal, double width, double height, _Out_ int* charColumns, _Out_ int* charRows);
+    __declspec(dllexport) HRESULT _stdcall TriggerResize(void* terminal, double width, double height, _Out_ int* charColumns, _Out_ int* charRows);
```
Comment (ZoeyR) [inline src/cascadia/PublicTerminalCore/HwndTerminal.hpp]: looks like these got an extra space preceeding them somehow

File: src/cascadia/PublicTerminalCore/HwndTerminal.cpp (hunk 1 above)
Comment (javierdlg) [reply src/cascadia/PublicTerminalCore/HwndTerminal.cpp]: Nice catch, I've gone ahead and changed this to send the rows and columns instead.

File: src/cascadia/PublicTerminalCore/HwndTerminal.hpp (hunk 2 above)
Comment (javierdlg) [reply src/cascadia/PublicTerminalCore/HwndTerminal.hpp]: Can't see this locally, and if I replace the spacing with the other methods it still doesn't show as a change. Might be my commit changing the spacing...

Comment (ZoeyR) [reply src/cascadia/PublicTerminalCore/HwndTerminal.hpp]: I think its a tabs/spaces issue.

Comment (javierdlg) [reply src/cascadia/PublicTerminalCore/HwndTerminal.hpp]: VS is acting up, I had to just copy paste on all lines :/

**Pull request merged**